src/Microsoft.DotNet.Cli.Utils/ProjectContextExtensions.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+using Microsoft.DotNet.ProjectModel;
+using Microsoft.DotNet.ProjectModel.Graph;
+using Microsoft.DotNet.Tools.Common;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    internal static class ProjectContextExtensions
+    { 
+        internal static void MakeRunnable(this ProjectContext context, string outputPath, string configuration)
+        {
+            context
+                .ProjectFile
+                .Files
+                .GetCopyToOutputFiles()
+                .StructuredCopyTo(context.ProjectDirectory, outputPath)
+                .RemoveAttribute(FileAttributes.ReadOnly);
+
+            var exporter = context.CreateExporter(configuration);
+
+            if (context.TargetFramework.IsDesktop())
+            {
+                exporter
+                    .GetDependencies()
+                    .SelectMany(e => e.RuntimeAssets())
+                    .CopyTo(outputPath);
+
+                GenerateBindingRedirects(context, outputPath, configuration);
+            }
+            else
+            {
+                exporter
+                    .GetDependencies(LibraryType.Package)
+                    .WriteDepsTo(Path.Combine(outputPath, context.ProjectFile.Name + ".deps"));
```

Comment:
Fixed globally, though moved to FileNameSuffixes.cs instead the Contsants super class [dumping ground? :P]